### PR TITLE
[Organization settings] Bisect General settings

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
## Description

Related & For #19788 (see base branch)

Bisects the General settings to General + Policies 

## Related Issue(s)

Fixes OPM-608

Fixes OPM-607

Fixes OPM-611

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - siddhant-ofe0cb46cdd</li>
	<li><b>🔗 URL</b> - <a href="https://siddhant-ofe0cb46cdd.preview.gitpod-dev.com/workspaces" target="_blank">siddhant-ofe0cb46cdd.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - siddhant-opm-607-general-gha.25546</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-siddhant-ofe0cb46cdd%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-dev-preview" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [x] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [x] /werft preemptible
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

File: components/dashboard/src/components/podkit/buttons/LinkButton.tsx
Comment (Siddhant-K-code) [inline components/dashboard/src/components/podkit/buttons/LinkButton.tsx:1]: Podkit component change to introduce & Fix External URL behavior

File: components/dashboard/src/components/PillLabel.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ export type PillType = "info" | "warn" | "success" | "neutral";
 
 const PillClsMap: Record<PillType, string> = {
     info: "bg-blue-50 text-blue-500 dark:bg-blue-500 dark:text-blue-100",
-    warn: "bg-orange-100 text-orange-700 dark:bg-orange-600 dark:text-orange-100",
+    warn: "bg-kumquat-light text-pk-content-secondary",
```
Comment (Siddhant-K-code) [inline components/dashboard/src/components/PillLabel.tsx:11]: Align with the new component library

File: components/dashboard/src/teams/TeamPolicies.tsx
Comment (Siddhant-K-code) [inline components/dashboard/src/teams/TeamPolicies.tsx:1]: Most of the code is same as before (it just the file splits)

File: components/dashboard/src/teams/TeamPolicies.tsx [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+                >
+                    Dcoumentation
+                </LinkButton>
+                <LinkButton variant="secondary" href="https://www.gitpod.io/docs/enterprise" isExternalUrl={true}>
+                    Learn more about Enterprise
+                </LinkButton>
+            </div>
+        </ConfigurationSettingsField>
+    );
+};
```
Comment (Siddhant-K-code) [inline components/dashboard/src/teams/TeamPolicies.tsx:304]: New Addition

File: components/dashboard/src/teams/TeamPolicies.tsx [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+                        />
+                    </ConfigurationSettingsField>
+
+                    <OrgWorkspaceClassesOptions
+                        isOwner={isOwner}
+                        settings={settings}
+                        handleUpdateTeamSettings={handleUpdateTeamSettings}
+                    />
+
+                    {isGitpodIo() && <WorkspaceClassesEnterpriseCallout />}
```
Comment (Siddhant-K-code) [inline components/dashboard/src/teams/TeamPolicies.tsx:102]: New addition

**Comment by corygrunk** [comment]
Can you apply this tailwindcss to the "Enterprise" tag `px-3 py-0 text-sm uppercase rounded-xl bg-kumquat-light text-pk-content-secondary !py-0.5 border`?

**Comment by corygrunk** [comment]
Also, the Enterprise tag should be `text-xs`

**Pull request merged**